Makefile

```diff
@@ -249,7 +250,7 @@ docs-serve: docs
 
 .PHONY: docs-linkcheck
 docs-linkcheck: /usr/local/bin/lychee
-	lychee --exclude-path=CHANGELOG.md --exclude-mail *.md
+	lychee --exclude-path=CHANGELOG.md --exclude-mail *.md --include "https://github.com/numaproj/*" $(shell find ./test -type f) $(wildcard ./docs/*.md)
```

Comment:
Does this make sense? The CI is checking remote files instead of local files (with changes)?

Reply:
Are you referring to the --include specification?

```
--include <INCLUDE>
          URLs to check (supports regex).
```

It basically specifies that lychee is going to scan all files under ./test and ./docs, and look for every url that starts with "https://github.com/numaproj/". Then lychee will ping the URLs to check that the link is working and return errors if not.

Reply:
The PR is mainly to make sure when we update SDK packages with path change, we update the example links in numaflow accordingly.

Reply:
I see. But is it limited to only check the links starting with "https://github.com/numaproj/"? which means ignore that links even they are broken?

Reply:
Yes. Mainly because if I don't add this prefix, it will scan URLs like "localhost:8080" which is naturally broken and also other links like "https://v1-18.docs.kubernetes.io/docs/reference/generated/*", "https://kubernetes.io/docs/*". The number of links can be very high and I noticed lychee calls being throttled. So I decided to just check numaproj links, for now.